### PR TITLE
small changes to ui for login/register

### PR DESCRIPTION
**Pull request by williambertrand**
### PR Description
- Add a way to get back to Login from register
- Tweaks to button and font sizes
    - Reduced a lot of the font sizes cuz they looked pretty massive on device
- Adds `Linkbutton`
    - Generally I like for buttons that take you to a new screen to look different than buttons that perform an action on the screen, so added this for going from Login -> Signup and vice versa

### Captures 
| Log in | Sign up |
| --- | --- |
|  |

### CODE REVIEW

File: components/AppContext.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
 
+type ColorScheme = 'light' | 'dark'
+
 const AppContext = React.createContext({
   token: "" as string | null | undefined,
-  scheme: "",
+  scheme: 'light' as ColorScheme,
```
Comment (williambertrand) [inline components/AppContext.tsx:7]: Should default to a valid colorsheme instead of `""`

File: screens/RegisterScreen.tsx [hunk 2]
```diff
@@ -61,15 +57,27 @@ export default function RegisterScreen({ navigation }) {
...
+        <Text style={styles.body}>Already have an account?</Text>
+        <LinkButton
+        darkColor="#fff"
+        onPress={() => {
+          navigation.navigate("Login");
+        }}
+        title='Log in'
+      />
+
+      </View>
```
Comment (williambertrand) [inline screens/RegisterScreen.tsx:79]: There was no way to get back to the login screen

File: components/AppContext.tsx (hunk 1 above)
Comment (probicheaux) [reply components/AppContext.tsx:7]: oh cool thx

File: screens/RegisterScreen.tsx (hunk 2 above)
Comment (probicheaux) [reply screens/RegisterScreen.tsx:79]: true

**Review by probicheaux — APPROVED** [review]
lgtm boss

**Pull request merged**